examples/multiphysics/warming_up_viscous_fluid/warming_up_viscous_fluid.prm

```diff
@@ -0,0 +1,149 @@
+#---------------------------------------------------
+# FEM
+#---------------------------------------------------
+subsection FEM
+    set velocity order        = 1
+    set pressure order        = 1
+    set temperature order     = 2
+end
+
+#---------------------------------------------------
+# Timer
+#---------------------------------------------------
+subsection timer
+    set type    = none
+end
+
+#---------------------------------------------------
+# Initial condition
+#---------------------------------------------------
+subsection initial conditions
+    set type = nodal
+    subsection uvwp
+            set Function expression = 0; 0; 0
+    end
+end
+
+#---------------------------------------------------
+# Physical Properties
+#---------------------------------------------------
+subsection physical properties
+  set density              = 0.9
+  set kinematic viscosity    = 0.5
+  set thermal conductivity = 0.12
+# water = 1 density, 0.01 viscosity, 0.59 conductivity
+# oil = 0.9 density, 0.5 viscosity, 0.12 conductivity
+end
+
+#---------------------------------------------------
+# Mesh
+#---------------------------------------------------
+subsection mesh
+  set type = dealii
+  set grid type = hyper_rectangle
+  set grid arguments = 0, 0 : 0.5, 1 : true
+  set initial refinement = 4
+end
+
+#---------------------------------------------------
+# Multiphysics
+#---------------------------------------------------
+subsection multiphysics
+  set heat transfer = true
+  set viscous dissipation = true
+end
+
+# --------------------------------------------------
+# Mesh Adaptation Control
+#---------------------------------------------------
+subsection mesh adaptation
+  set type                    = none
+end
+
+# --------------------------------------------------
+# Analytical Solution
+#---------------------------------------------------
+subsection analytical solution
+  set enable                 = true
+  set verbosity = verbose
+    subsection uvwp
+            set Function expression =  0 ; 0 ; 0
+    end
+    subsection temperature
+	  set Function constants = rho=0.9, nu=0.5, K=0.12, Tw=80, v=2, B=0.5
+          set Function expression = Tw+(((rho*nu)*v*v)/(2*K))*(1-(x/B)*(x/B))
+    end
+end
+
+# --------------------------------------------------
+# Boundary Conditions
+#---------------------------------------------------
+subsection boundary conditions
+  set number                  = 2
+    subsection bc 0
+    set id = 0
+        set type              = noslip
+    end
+    subsection bc 1
+    set id = 1
+        set type              = function
+        subsection u
+            set Function expression = 0
+        end
+        subsection v
+            set Function expression = 2
+        end
+    end
+end
+
+subsection boundary conditions heat transfer
+  set number                  = 2
+    subsection bc 0
+    set id = 0
+        set type          = convection
+	set h             = 0
+	set Tinf	  = 0
+    end
```

Comment:
Why a convection bc with h=0 & T_inf=0?

Reply:
For `bc 0` we have a thermic insulation condition, so `h=0` defines a "no heat transfer" condition. However `T_inf` does not have to be given. If you find this misleading we can remove it!

Reply:
Well it's just that you could have specified no boundary condition and it would have been the same (a neumann instead of a Robin with h=0), but it's equivalent, so let's leave it at that.